### DISCUSSION TITLE
Support for Device Details

### DISCUSSION THREAD

**Discussion by ShivMSense** [getsentry/sentry-dart#1904] [Q&A]
Hi
Does Sentry Flutter SDK capture the device details automatically or I have to enable it somewhere in my code or Project Settings?
In my case, this placeholder is shown instead of device details for all the errors/events logged:

<img width="338" alt="Screenshot 2024-03-01 at 11 21 47 AM" src="https://github.com/getsentry/sentry-dart/assets/122361459/a1e3dedf-4593-4706-a1aa-b995e7799676">

**Comment by buenaflor** [comment]
Hey sorry for the late response, is this still an issue?

did you use `SentryFlutter.init`?
